src/Routes/bus.js

```diff
@@ -8,7 +8,6 @@ const router=express.Router()
 
 router.use(cors())
       
-
 router.route("/admin/bus/createbus")
```

Comment:
Use kebab-case or dash-case for naming URL paths. And also, here in this case, /bus/createBus. Since bus is there already, I would recommend not to use Bus in createBus.

Reply:
changed the action item